app/configdialog.cpp

```diff
@@ -82,8 +82,9 @@ void ConfigDialog::writeSettings()
 
 void ConfigDialog::setTranslatedHighlightTypeNames(const QStringList &typeNames)
 {
-    Q_FOREACH (const QString &typeName, typeNames)
```

Comment:
I know it is not related, but what you think of using auto.

Instead of types use: `const auto&`

Reply:
Yeah, sure, that sounds good to me! I'll change them in these for loops.

Reply:
Done in [8086ab3](https://github.com/fhackenberger/ktikz/pull/64/commits/8086ab3e35ae03e298a25d1f62c9c51093029698).